scroll-kit.coffee

```diff
@@ -330,6 +339,9 @@ initialize_sticky = (node) ->
   node.isFixed = data.fixed or (el.css('position') is 'fixed')
   node.placeholder = placeholder(node) if update_sticky(node)
 
+  if node.data.fit
+    node.el.on('DOMMouseScroll mousewheel', preventScroll)
```

Comment:
I'm not sure if attaching this event per-sticky is a good idea, my suggestion is attaching only a global handler for the whole body, then, identify which sticky is within this handler and prevent, etc.